docs/source/upgrade.md

```diff
@@ -2,6 +2,24 @@
 
 This pages describes Scylla Operator upgrade procedures.
 
+## Upgrade via Helm
```

Comment:
Add few sentences explaining that helm can't handle the CRD update, possibly links

Reply:
done